stestr/commands/run.py

```diff
@@ -206,6 +207,21 @@ def run_tests():
             result = run_tests()
```

Comment:
If this first time run_tests() fails with using subunit, we miss the failure. Do I miss something?